kasa/smart/modules/batterysensor.py

```diff
@@ -55,4 +55,6 @@ def battery(self) -> int:
     @property
     def battery_low(self) -> bool:
         """Return True if battery is low."""
-        return self._device.sys_info["at_low_battery"]
+        return self._device.sys_info.get(
+            "at_low_battery", self._device.sys_info.get("is_low")
+        )
```

Comment:
I think we should make sure not to return None or change the type hint.

```suggestion
        if batt_low := self._device.sys_info.get("at_low_battery", self._device.sys_info.get("is_low")):
            return batt_low
        _LOGGER.debug("Could not get a proper value for low battery")  # log once only perhaps?
        return False
```

Reply:
Or perhaps just avoid creating the feature if none of the known keys are available, and raise an exception inside the method if no info is found and the getter is accessed?

Reply:
Yes makes sense, so the attribute can be a `FeatureAttribute`

Reply:
Done.